auv_sim/auv_sim_bringup/launch/start_gazebo.launch

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0"?>
 <launch>
+  <env name="GAZEBO_MODEL_PATH" value="$(find auv_sim_description)/models:${GAZEBO_MODEL_PATH}"/>
```

Comment:
Set the gazebo model path correctly and automatically for convenience. Works as intended 👍

Reply:
Is this required? The package.xml already has export section and should be handling this automatically. Can you clean your build, remove this, and rely on the package.xml export and see if problem persists?

Reply:
You are referring to;

```xml
    <gazebo_ros gazebo_model_path="${prefix}/meshes" />
    <gazebo_ros gazebo_model_path="${prefix}/urdf" />
    <gazebo_ros plugin_path="${preffix}/lib" />
    <gazebo_ros gazebo_media_path="${preffix}" />
```
I tested these, and also tried different methods utilizing package.xml, but they don't seem to work. Best way I could find was setting the environment variable at .launch file. 

Here is a quote from ChatGPT explaining why package.xml probably won't work:

> Unfortunately, using just the package.xml file to set environment variables at runtime is not a standard approach in ROS 1. In ROS 1 (with catkin), environment variables are typically set through environment hooks, rather than directly through package.xml.
> Why package.xml doesn’t work directly for environment variables:
> In ROS 1, package.xml describes package metadata (name, version, dependencies, etc.) but does not provide a mechanism to dynamically modify environment variables at runtime after the user sources their workspace.